clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -990,12 +990,12 @@ private PartitionRecords parseCompletedFetch(CompletedFetch completedFetch) {
                     throw new OffsetOutOfRangeException(Collections.singletonMap(tp, fetchOffset));
                 }
             } else if (error == Errors.TOPIC_AUTHORIZATION_FAILED) {
-                log.warn("Not authorized to read from topic {}.", tp.topic());
+                log.warn("Not authorized to read from topic {} when fetching for partition {}.", tp.topic(), tp);
```

Comment:
Isn't just `tp` sufficient, as it is converted to string as `{topic}-{partition}`?

Reply:
yes, it is very sufficient. since im unclear on what constitutes an "api" i was trying not to change existing log formats. will update.